adapter-base-quarkus/src/main/resources/META-INF/native-image/org.eclipse.hono/hono-adapter-base-quarkus/native-image.properties

```diff
@@ -8,4 +8,4 @@
 # http://www.eclipse.org/legal/epl-2.0
 #
 # SPDX-License-Identifier: EPL-2.0
-Args = -H:ReflectionConfigurationResources=${.}/reflection-config.json, --initialize-at-run-time=io.netty.internal.tcnative.SSL
+Args = -H:ReflectionConfigurationResources=${.}/reflection-config.json, -H:+AllowIncompleteClasspath --initialize-at-run-time=io.netty.internal.tcnative.SSL
```

Comment:
why do we need to allow an incomplete classpath here?

Reply:
Otherwise there build be such an error when building the native image without using the `jaeger` maven profile:
````
Error: Error parsing reflection configuration in jar:file:/project/lib/org.eclipse.hono.hono-adapter-base-quarkus-1.9.0-SNAPSHOT.jar!/META-INF/native-image/org.eclipse.hono/hono-adapter-base-quarkus/reflection-config.json:
Could not resolve io.jaegertracing.internal.samplers.http.OperationSamplingParameters for reflection configuration. To allow unresolvable reflection configuration, use option -H:+AllowIncompleteClasspath
````

Reply:
I see. On the other hand, it will prevent the build from detecting missing classes that are being referenced during build time, i.e. such errors will only become apparent during runtime. But that is probably not such a big issue as most problems that we are facing are related to missing classes that are referred to via reflection only anyway ....

Reply:
I agree that it's better to not have the AllowIncompleteClasspath option in there.
I've adapted the commit now so that it isn't needed anymore.